.travis.yml

```diff
@@ -59,4 +67,29 @@ jobs:
     - MB_PYTHON_VERSION=3.10
     - MB_ML_VER=2010
     - PLAT=i686
+  - arch: arm64
+    os: linux
+    services: docker
+    env:
+    - MB_PYTHON_VERSION=3.10
+    - MB_ML_VER=2014
+    - PLAT=aarch64
+    - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.11
+    - MB_ML_VER=2014
```

Comment:
When we first added support for wheel builds for ARM platforms it was [noted](https://github.com/datastax/python-driver-wheels/pull/4#issuecomment-1102833548) that manylinux1 images didn't include a Python 3.10 binary; to get that we had to go up to manylinux 2010.  Local testing showed that the current manylinux 2010 image doesn't contain Python 3.11 binaries, while the manylinux 2014 image does.